apps/frontend/src/shared-research/__tests__/ResearchOutput.test.tsx

```diff
@@ -69,27 +69,27 @@ const renderComponent = async () => {
   return result;
 };
 
-describe('a proposal research output', () => {
+describe('a Grant Document research output', () => {
   it('renders with its teams', async () => {
```

Comment:
i suppose this test should show the difference in rendering proposals or grant documents and other types - i think for the time being you should show that both Grant Documents and Proposal types are rendered differently (i think the test technically show that only the grant document one is different)